### ISSUE TITLE
Accessing pod logs tail

### ISSUE THREAD

**Issue by brobichaud** [tintoy/dotnet-kube-client#65]
I'm trying to get the last X lines of a pods logs, something like what the k8s dashboard does when viewing logs. I've experimented with the two obvious ways of acquiring the logs:

IPodClientV1.Logs()
IPodClientV1.StreamLogs()

And see they both support a limitBytes option, but both seem to start with the beginning of the logs and not the end. Some of my pods have very large logs so even if I specify a reasonable limitBytes of say 100KB I only get the very beginning of my logs.

The k8s dashboard seems to get the logs very quickly even when I have a pod with many MB of logs, so it seems like it has an intelligent way to get the end of the log stream. Any ideas how I can replicate this with this api?

**Comment by tintoy** [comment]
Hmm, interesting question - if you run your `kubectl` command with `—v=10` (note the double dash) it should log the API calls it is making which may be of some assistance. I’ll take a look tonight myself and see if anything stands out...

**Comment by tintoy** [comment]
I suspect it haven’t exposed the query parameters needed to scan from the end of the log yet - the API docs list a few promising candidates.

**Comment by brobichaud** [comment]
Yeah I just used this command line: kc logs --tail=20 podname -v=10

And it shows the api call of:
 GET /api/v1/namespaces/default/pods/dev-madras-aggregator-97988547-vsrnh/log?tailLines=20

I'm assuming the dashboard is using this method or something similar.

**Comment by tintoy** [comment]
Yep, pretty sure I can expose it on both :)

**Comment by tintoy** [comment]
Will add it this weekend and publish a new set of packages :)

**Comment by tintoy** [comment]
Published [v2.2.6](https://github.com/tintoy/dotnet-kube-client/releases/tag/v2.2.6).

**Comment by felixfbecker** [comment]
Shipped in PSKubectl, can verify it works :)

**Comment by brobichaud** [comment]
Sweet! I just upgraded to the v2.2.6 nuget and the Logs() method is perfecto! Thanks a ton for the super quick turn-around @tintoy.

I'm experimenting with the StreamLogs() method and am clearly using it wrong. I can get the first pull of data but am not seeing updates even though I've implemented IObserver<string>. OnNext is never called. Then it stops working at all after I've called this method a couple of times. Has anyone used StreamLogs() successfully?
